store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -1128,8 +1136,9 @@ func (mvcc *MVCCLevelDB) CheckTxnStatus(primaryKey []byte, lockTS, callerStartTS
 	// written before the primary lock.
 
 	if rollbackIfNotExist {
+		// Write rollback record, but not delete the lock
```

Comment:
Why not delete the lock, to keep consistent with TiKV?

Reply:
There may exist one lock on the primary key, and this lock belongs to other transactions, like the transaction which raises the resolve action itself, we've already checked the lock for the primary  key and `lockTS`

Reply:
could we add this explanation as code comments, other code readers may have had the same question in mind

Reply:
> There may exist one lock on the primary key, and this lock belongs to other transactions

And we're rolling back it.

Reply:
Finally I understand here.